testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/tableProperties.tests.js

```diff
@@ -210,6 +218,40 @@ module('Table properties forms', {
             assert.strictEqual($tableElement.css('textAlign'), 'right', 'text align is applied');
         });
 
+        test('Passing backgroundColor & borderColor from empty selected cells to colorBox', function(assert) {
```

Comment:
Let's answer the question "how should it work?"
For example, ""
```suggestion
        test('cell backgroundColor should be passed to colorBox as a default during editing', function(assert) {
```